src/main/java/org/springframework/social/bitbucket/api/impl/BitBucketTemplate.java

```diff
@@ -54,6 +57,11 @@ public final RepoOperations repoOperations() {
     }
 
     @Override
+    public final GroupsOperations groupsOperations() {
```

Comment:
Mayby not here. Interface already introduces methods without "get". It will break applications that already uses this lib in 2.0.0-SNAPSHOT version.